project/UpdateEclipseClasspaths.scala

```diff
@@ -63,6 +65,7 @@ object UpdateEclipseClasspaths extends App {
     //
     val s = System.getenv("DAFFODIL_HOME")
     assert(s ne null, "$DAFFODIL_HOME undefined")
+    System.err.println("DAFFODIL_HOME environment variable: " + s)
```

Comment:
Probably these should all just be println. Not system.err.println.

Reply:
I'm wondering if we should change this to use sbt logging rather than println? Then we can give appropriate info/error diagnostics, so errors will really jump out? It might take a bit of refactoring to passing in the ``stream.log`` from sbt. And it more greatly ties this tool to sbt, but that doesn't seem unreasonable. 

Also related, I just opened [DAFFODIL-2465](https://issues.apache.org/jira/browse/DAFFODIL-2465). If it's okay to add assume this is always run via sbt, then we can also easily get access to baseDirectory.value and I think get rid of DAFFODIL_HOME completely.